JitsiConferenceEventManager.js

```diff
@@ -85,11 +75,14 @@ JitsiConferenceEventManager.prototype.setupChatRoomListeners = function() {
     });
 
     chatRoom.addListener(XMPPEvents.AUDIO_MUTED_BY_FOCUS,
-        value => {
-            Statistics.analytics.sendEvent(REMOTELY_MUTED);
+        () => {
+            // TODO: Add a way to differentiate between commands which caused
+            // us to mute and those that did not change our state (i.e. we were
+            // already muted).
+            Statistics.sendAnalytics(createRemotelyMutedEvent());
```

Comment:
I noticed sendAnalytics takes in eventName and attributes but that calls into analytics.sendEvent(eventName, properties). In some cases you're passing eventName as an object. Can you confirm that analytics.sendEvent properly handle both strings and objects as eventName (and maybe where I can look for its definition)?

Reply:
Nevermind...I got lost while searching in the changset. It does handle it fine.